.changeset/strong-flies-thank.md

```diff
@@ -0,0 +1,5 @@
+---
+"@khanacademy/perseus": major
```

Comment:
This feels almost like overkill, but we're removing types from the exported package, so that constitutes a major version bump.